### PR TITLE
Nikis Stanage docs

### PR DESCRIPTION
**Pull request by nikishe**
None tabbed docs

### CODE REVIEW

File: stanagepilot/software/apps/gmp.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    module load GMP/6.2.0-GCCcore-10.2.0                       
+    module load GMP/6.2.0-GCCcore-9.3.0                       
+    module load GMP/6.1.2-GCCcore-8.3.0
+    module load GMP/6.1.2-GCCcore-7.3.0   
+
+
+Installation notes
+------------------
+
+GnuMP was installed using Easybuild 4.7.0, build details can be found in $EBROOTGMP/easybuild with the module loaded.
```
Comment (Carldkennedy) [inline stanagepilot/software/apps/gmp.rst:34]:
```suggestion
GnuMP was installed using Easybuild 4.7.0, build details can be found in ``$EBROOTGMP/easybuild`` with the module loaded.
```

File: stanagepilot/software/libs/libsndfile.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* Usable on Unix, Win32, MacOS and others.
+* On the fly format conversion, including endian-ness swapping, type conversion and bitwidth scaling.
+* Optional normalisation when reading floating point data from files containing integer data.
+* Ability to open files in read/write mode.
+* The ability to write the file header without closing the file (only on files open for write or read/write).
+* Ability to query the library about all supported formats and retrieve text strings describing each format. 
+
+Usage
+-----
+To make this library available, run the following module commands
```
Comment (Carldkennedy) [inline stanagepilot/software/libs/libsndfile.rst:38]:
```suggestion
To make this library available, run one of the following module commands:
```

File: stanagepilot/software/libs/libunistring.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      module load libunistring/0.9.10-GCCcore-10.3.0                     
+      module load libunistring/1.0-GCCcore-11.3.0  
+
+This correctly populates the environment variables ``LD_LIBRARY_PATH``, ``LIBRARY_PATH`` and ``CPATH``.
+
+Installation Notes
+------------------
+This section is primarily for administrators of the system.
+
+GnuMP was installed using Easybuild 4.7.0, build details can be found in $EBROOTGMP/easybuild with the module loaded.
```
Comment (Carldkennedy) [inline stanagepilot/software/libs/libunistring.rst:39]:
```suggestion
GnuMP was installed using Easybuild 4.7.0, build details can be found in ``$EBROOTGMP/easybuild`` with the module loaded.
```

File: stanagepilot/software/libs/udunits.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  module load UDUNITS/2.2.28-GCCcore-11.2.0                
+  module load UDUNITS/2.2.28-GCCcore-11.3.0
+  
+--------
+
+Installation notes
+------------------
+This section is primarily for administrators of the system. 
+
+udunits was installed using Easybuild 4.7.0, build details can be found in $EBROOTGMP/easybuild with the module loaded.
```
Comment (Carldkennedy) [inline stanagepilot/software/libs/udunits.rst:44]:
```suggestion
udunits was installed using Easybuild 4.7.0, build details can be found in ``$EBROOTGMP/easybuild`` with the module loaded.
```

File: stanagepilot/software/libs/udunits.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Testing
+-------
+
+    1. load Module
+
+    2. Run “udunits2“
+
+    3. For my test i converted 5km to miles , getting the following results:
```
Comment (Carldkennedy) [inline stanagepilot/software/libs/udunits.rst:53]:
```suggestion
    3. For this test we convert 5km into miles, which produces the following results:
```

**Review by Carldkennedy — COMMENTED** [review]
LGTM. Suggested a few changes

File: stanagepilot/software/apps/gmp.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+GnuMP is a portable library written in C for arbitrary precision arithmetic on integers, rational numbers, and floating-point numbers. It aims to provide the fastest possible arithmetic for all applications that need higher precision than is directly supported by the basic C types. 
+
+
+Usage
+-----
+
+GnuMP can be activated as follows:
+
+.. code-block:: console
```
Comment (jkwmoore) [inline stanagepilot/software/apps/gmp.rst:20]:
```suggestion
.. code-block::
```

I prefer the highlighting on numbers:
 
image

Someone merge if they agre / close if they don't.

File: stanagepilot/software/apps/index.rst [hunk 7]
```diff
@@ -8,7 +8,9 @@ Applications on Stanage
     :glob:
 
     ansys/index
+    gmp
     openfoam
+    python
     vasp
     ncdu
     mathematica
```
Comment (jkwmoore) [inline stanagepilot/software/apps/index.rst:16]:
```suggestion
    ./*
```

Amend to grab all at the same directory level.

File: stanagepilot/software/apps/python.rst [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+Python
+======
+
+.. sidebar:: Python
+    
+    :Support Level: Core
+    :URL: https://python.org
+
+
+This page documents the "anaconda" installation on Bessemer. This is the
```
Comment (jkwmoore) [inline stanagepilot/software/apps/python.rst:12]:
```suggestion
This page documents the "anaconda" installation on Stanage. This is the
```

**Pull request merged**